### PR TITLE
Link moto in about_us to anthem.html

### LINKED ISSUE #57: Link "moto" to webpage
**Is your feature request related to a problem?**
In about_us.html, under bullets "moto" hasn't been linked to a webpage.

**Describe the solution you'd like**
Suggesting it would be suitable to link that to anthem.html page

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by Gimhan-minion**
## Purpose
The purpose of this PR is to fix #57  

## Goals
To link "moto" in about_us.html to a webpage

## Approach
Linked "moto" to anthem.html

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: about_us.html [hunk 1]
```diff
@@ -205,11 +205,11 @@ <h4>Who We Are</h4>
...
 							<ul class="list-block">
-                                <li><a href="#"><i class="icon-note"> </i>Our Moto</a></li>                                
-                                <li><a href="#"><i class="icon-note"> </i>Vision & Mission</a></li>
-								<li><a href="anthem.html"><i class="icon-note"> </i>College Crest</a></li>
-								<li><a href="#"><i class="icon-note"> </i>College Flag</a></li>
-                                <li><a href="anthem.html"><i class="icon-note"> </i>Anthem</a></li>
+                                <li><a href="anthem.html"><i class="icon-note"></i>Our Moto</a></li>
+                                <li><a href="#"><i class="icon-note"></i>Vision & Mission</a></li>
+								<li><a href="anthem.html"><i class="icon-note"></i>College Crest</a></li>
+								<li><a href="#"><i class="icon-note"></i>College Flag</a></li>
```
Comment (Piumal1999) [inline about_us.html:211]: Remove extra tab spaces

Comment (Gimhan-minion) [reply about_us.html:211]: > Remove extra tab spaces

Done

**Comment by jayasanka-sack** [comment]
Good job @Gimhan-minion ! 
Could you please change the PR title with a descriptive one?

**Comment by Gimhan-minion** [comment]
> Good job @Gimhan-minion !
> Could you please change the PR title with a descriptive one?

Done

**Comment by jayasanka-sack** [comment]
Btw, the preview link is wrong. It publishes ur master branch.

File: about_us.html [hunk 2]
```diff
@@ -205,11 +205,11 @@ <h4>Who We Are</h4>
 							<p>St. Anthony's College is a 1A,B school managed under the Provincial Council of Central Province. St. Anthony's College provides education to students from grade 1 to 13. The lecturing is done in all three media; Sinhala, Tamil and Bilingual. Admissions are in accordance with the mechanisms employed by the government in taking Children to government schools.</p>
                                 <hr>
 							<ul class="list-block">
-                                <li><a href="#"><i class="icon-note"> </i>Our Moto</a></li>                                
-                                <li><a href="#"><i class="icon-note"> </i>Vision & Mission</a></li>
-								<li><a href="anthem.html"><i class="icon-note"> </i>College Crest</a></li>
-								<li><a href="#"><i class="icon-note"> </i>College Flag</a></li>
-                                <li><a href="anthem.html"><i class="icon-note"> </i>Anthem</a></li>
+                                				<li><a href="anthem.html"><i class="icon-note"></i>Our Moto</a></li>
```
Comment (jayasanka-sack) [inline about_us.html:208]: Why this anthem is linked with the moto?

Comment (Piumal1999) [reply about_us.html:208]: The moto is displayed on the top section of the anthem page. I think we should change the name of the page.

Comment (Gimhan-minion) [reply about_us.html:208]: Anything else to be changed? @jayasanka-sack @Piumal1999

Comment (jayasanka-sack) [reply about_us.html:208]: Let's rename anthem.html with another suitable name. We can do it with another PR.

**Pull request merged**